src/qililab/analog/one_qubit_transpiler.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 Qilimanjaro Quantum Tech
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Callable
+
+from qililab.analog import Parameter
+
+
+class Qubit2LevelTranspiler:  # pylint: disable=too-few-public-methods
+    """Implementation of the transpiler for the 2 level qubit. This is done mainly by inverting the same functions
+    used int he single_qubit_2level emulator model.
+
+    Args:
+        eps_model (Callable): epsilon model
+        delta_model (Callable): delta model
+        qubitData (DataClass): dataclass containing info about the physical parameters of the qubit. Defaults to None.
+
+    Calling an instance of this class returns the fluxes phix, phiz for some given Delta, epsilon.
+    """
+
+    def __init__(self, epsilon_model: Callable, delta_model: Callable):
+        """Init method. See class description for more details"""
+
+        self.delta_model = delta_model
+        self.epsilon_model = epsilon_model
+
+        # Magnetic energy bias (epsilon)
+        self.epsilon = Parameter(name="epsilon", set_method=self._set_epsilon)
+        # Qubit gap (delta)
+        self.delta = Parameter(name="delta", set_method=self._set_delta)
+        # flux parameters
+        self.phiz = Parameter(name="phiz")
+        self.phix = Parameter(name="phix")
+
+    def __call__(self, delta: float, epsilon: float) -> tuple[Any, Any]:
+        """Transpiles Delta and Epsilon to phix, phiz"""
+        self.delta(delta)
+        self.epsilon(epsilon)
+        return self.phix(), self.phiz()  # type: ignore[func-returns-value]
```

Comment:
Just out of curiosity, what was the mypy issue before you included the type ignore here?

Reply:
It was complaining about `parameter()` not returning a value (function call does not return a value),  Now that I think about it maybe the issue was that I hadn't added type return to `parameter.get()`, I think it should work fine without the ignore now (trying this in the next commit)

Reply:
Yep, sounds right what you are saying.